test/e2e-tests.sh

```diff
@@ -17,7 +17,7 @@
 source $(dirname $0)/e2e-common.sh
 
 # Script entry point.
-initialize $@  --skip-istio-addon
```

Comment:
can you drop the `--skip-istio-addon` while we're here

it's not needed anymore